specificatie/immigratie.yaml

```diff
@@ -0,0 +1,49 @@
+openapi: 3.0.3
+info:
+  title: Immigratie definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    IndicatieVestigingVanuitBuitenland:
+      type: boolean
+      description: |
+        Geeft aan dat de persoon zich vanuit het buitenland heeft ingeschreven.
+    GbaImmigratie:
+      type: object
+      description: |
+        Gegevens over het het immigratie van een persoon.
+          * **landVanWaarIngeschreven** : het land waar de persoon woonde voor (her)vestiging in Nederland.
+      properties:
+        datumVestigingInNederland:
+          $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+        landVanwaarIngeschreven:
+          $ref: 'common.yaml#/components/schemas/Waardetabel'
+        inOnderzoek:
```

Comment:
deze zit al in verblijfplaats. is dan dubbelop

Reply:
Immigratie is nu een groep die op hetzelfde niveau in de persoon zit als verblijfplaats. Ik kan dus Immigratie opvragen zonder de verbijfplaats. Dan wil ik ook inOnderzoek mee kunnen geven.

Reply:
@JohanBoer ik zie dit hetzelfde als naam (GbaNaamPersoon) en geboorte (GbaGeboorte). Die bevatten in de GbaPersoon ook geen inOnderzoek, omdat die al geleverd is in inOnderzoek op het niveau van de persoon.
Hetzelfde geldt hier ook. Je hebt aan één inOnderzoek van verblijfplaatsgegevens genoeg in de GbaPersoon. Anders zou je hetzelfde gegeven meerdere keren (dubbel) moeten leveren.
Ook voor gemeenteVanInschrijving en datumInschrijvingInGemeente geldt dat deze nu (dit PR) zijn opgenomen in de persoon, maar dat inOnderzoek hiervan zou worden geleverd in verblijfplaats.inOnderzoek

Dit alles geldt natuurlijk niet voor de Persoon, daarin is het wel opgesplitst.

Reply:
InOnderzoek verwijderd uit GbaImmigratie